sort/sort.go

```diff
@@ -13,6 +13,10 @@ import (
 	"sync"
 )
 
+const (
+	maxTmpFilesAllowed = 1000
```

Comment:
Why do we have this limit?

Reply:
I was running into some errors that I exceeded the maximum number of open files on my machine so I put a cap on it.